php.ini-development

```diff
@@ -996,8 +996,19 @@ cli_server.color = On
 ;intl.use_exceptions = 0
 
 [sqlite3]
+; Directory pointing to SQLite3 extensions
+; http://php.net/manual/en/sqlite3.loadextension.php
 ;sqlite3.extension_dir =
 
+; SQLite defensive mode flag (only available from SQLite 3.26+)
+; When the defensive flag is enabled, language features that allow ordinary
+; SQL to deliberately corrupt the database file are disabled. This forbids
+; writing directly to the schema, shadow tables (eg. FTS data tables), or
+; the sqlite_dbpage virtual table.
+; https://www.sqlite.org/c3ref/c_dbconfig_defensive.html
+; (for older SQLite versions, this flag has no use)
+sqlite3.defensive = 1
+
```

Comment:
minor comment, this is the default value, so have to be commented.

Fixed in 78c9a53df722132f9b2a1a4018eba6327438e695

Reply:
Thanks for catching this, @remicollet!

Reply:
Oh sure, thank you :)